schema/MULTISTATE_INDEPENDENT_EXPENDITURE.json

```diff
@@ -0,0 +1,1066 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/MULTISTATE_INDEPENDENT_EXPENDITURE.json",
+    "version": "8.3.0.1",
+    "title": "FEC MULTISTATE_INDEPENDENT_EXPENDITURE",
+    "description": "MULTISTATE_INDEPENDENT_EXPENDITURE",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_type_identifier",
+        "transaction_id",
+        "payee_street_1",
+        "payee_city",
+        "payee_state",
+        "payee_zip",
+        "election_code",
+        "expenditure_amount",
+        "calendar_ytd_per_election_office",
+        "aggregation_group",
+        "expenditure_purpose_descrip",
+        "support_oppose_code",
+        "so_candidate_id_number",
+        "so_candidate_last_name",
+        "so_candidate_first_name",
+        "so_candidate_office",
+        "so_candidate_state",
+        "completing_last_name",
+        "completing_first_name",
+        "date_signed"
+    ],
+    "properties": {
+        "form_type": {
+            "title": "FORM TYPE",
+            "description": "",
+            "const": "SE",
+            "examples": [
+                "SE"
+            ],
+            "fec_spec": {
+                "COL_SEQ": 1,
```

Comment:
Minor thing but might want to remove these COL_SEQ here and elsewhere

Reply:
Good catch. Will remove